packages/react-test-renderer/src/ReactTestRenderer.js

```diff
@@ -471,6 +474,17 @@ function create(
   getInstance(): React$Component<any, any> | PublicInstance | null,
   unstable_flushSync: typeof flushSync,
 } {
+  if (__DEV__) {
+    if (
+      enableReactTestRendererWarning === true &&
+      global.__REACT_DEVTOOLS_GLOBAL_HOOK__ == null
```

Comment:
Why are we checking this?

Reply:
Dev tools rely on RTR a bunch within their testing. This avoid the warning there which breaks tests in CI. I can remove for now since this PR isn't turning on the flag yet.